### PR TITLE
Unit Testing (EntraUserManager)

### PR DESCRIPTION
**Pull request by v-uansari**

### CODE REVIEW

File: test/module/Entra/Get-EntraUserManager.Tests.ps1 [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+    $scriptblock = {
+        # Write-Host "Mocking Get-EntraUserManager with parameters: $($args | ConvertTo-Json -Depth 3)"
+        return @(
+            [PSCustomObject]@{
+                Id                         = '00aa00aa-bb11-cc22-dd33-44ee44ee44ee'
+                ageGroup                   = $null
+                onPremisesLastSyncDateTime = $null
+                creationType               = $null
+                imAddresses                = @("miriamg@m365x99297270.onmicrosoft.com")
```
Comment (KenitoInc) [inline test/module/Entra/Get-EntraUserManager.Tests.ps1:18]: We should use a FAKE email address here

Comment (KenitoInc) [reply test/module/Entra/Get-EntraUserManager.Tests.ps1:18]: And all the places this is repeated

File: test/module/Entra/Get-EntraUserOwnedObject.Tests.ps1 [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+        return @{
+            value = @(
+                @{
+                    Id                    = "00aa00aa-bb11-cc22-dd33-44ee44ee44ee"
+                    applicationTemplateId = "00001111-aaaa-2222-bbbb-3333cccc4444"
+                    appId                 = "11112222-bbbb-3333-cccc-4444dddd5555"
+                    displayName           = "ToGraph_443DEM"
+                    signInAudience        = "AzureADMyOrg"
+                    publisherDomain       = "M365x99297270.onmicrosoft.com"
```
Comment (KenitoInc) [inline test/module/Entra/Get-EntraUserOwnedObject.Tests.ps1:21]: Let's use `contoso.com` domain

Comment (v-pughosh) [reply test/module/Entra/Get-EntraUserOwnedObject.Tests.ps1]: updated

File: test/module/Entra/Get-EntraUserManager.Tests.ps1 (hunk 1 above)
Comment (v-pughosh) [reply test/module/Entra/Get-EntraUserManager.Tests.ps1]: fixed and updated

**Review by KenitoInc — APPROVED** [review]

**Pull request merged**